tests/scripts/build-release.sh

```diff
@@ -0,0 +1,39 @@
+/bin/bash -e
+set -ex
+
+#############
+# VARIABLES #
+#############
+BRANCH_NAME=$1
+
+#############
+# FUNCTIONS #
+#############
+
+function  master_branch() {
+    build/run make -j$(nproc) VERSION=0 build.all
+    build/run make -j$(nproc) mod.check
+    build/run make -j$(nproc) -C build/release build BRANCH_NAME=${BRANCH_NAME}  GIT_API_TOKEN=${GIT_PSW}
+    git status &
+    git diff &
+    build/run make -j$(nproc) -C build/release publish BRANCH_NAME=${BRANCH_NAME} GIT_API_TOKEN=${GIT_PSW}
+    build/run make -j$(nproc) -C build/release promote TAG_WITH_SUFFIX=true BRANCH_NAME=master CHANNEL=master
```

Comment:
- Promote really only publishes the helm chart. It would be nice to rename this action so it's more clear, but that could be a separate PR.
- We should automatically `promote` 1) every master branch and 2) every tag. But we should *not* promote the release branch builds (unless tagged).